src/core/sys/posix/ucontext.d

```diff
@@ -1032,6 +1032,46 @@ else version (FreeBSD)
             ulong[8]        mc_spare;
         }
     }
+    else version (PPC)
+    {
+        alias uint   __register_t;
```

Comment:
This is the only thing that is different between PPC and PPC64, please alias it to `size_t` and merge the two into a `version (PPC_Any)`.